### PR TITLE
Release/1.6.0

### PR DESCRIPTION
**Pull request by repjarms**
### Summary

This is the PR to release version `1.6.0` of the Go SDK. It makes the following updates:

### Overall
- Docstring updates across methods

### AssistantV1
- Additional MessageInput model parameters

### AssistantV2
- Added the messageStateless operation and tests

### VisualRecognitionV4
- Added the getModel operation (currently only supports RSCNN for Android) and tests

### CODE REVIEW

File: assistantv1/assistant_v1_integration_test.go [hunk 1]
```diff
@@ -3,7 +3,7 @@
 package assistantv1_test
 
 /**
- * (C) Copyright IBM Corp. 2018, 2019.
+ * (C) Copyright IBM Corp. 2018, 2020.
```
Comment (mediumTaj) [inline assistantv1/assistant_v1_integration_test.go:6]: Did you run lint copyright dates?

File: comparecomplyv1/compare_comply_v1_integration_test.go [hunk 2]
```diff
@@ -3,7 +3,7 @@
 package comparecomplyv1_test
 
 /**
- * (C) Copyright IBM Corp. 2018, 2019.
+ * (C) Copyright IBM Corp. 2018, 2020.
```
Comment (mediumTaj) [inline comparecomplyv1/compare_comply_v1_integration_test.go:6]: same comment about lint copyright dates - i wont comment on these from this point

File: assistantv2/assistant_v2.go [hunk 3]
```diff
@@ -19,6 +19,7 @@ package assistantv2
 
 import (
 	"fmt"
+
```
Comment (mamoonraja) [inline assistantv2/assistant_v2.go:22]: is this being generated or unintentional?

Comment (repjarms) [reply assistantv2/assistant_v2.go:22]: The separation is intentional. Go formatter separates those for style purposes.

File: assistantv1/assistant_v1_integration_test.go (hunk 1 above)
Comment (repjarms) [reply assistantv1/assistant_v1_integration_test.go:6]: I ran it pre and post changes, but it looks like it missed a couple of these files

**Pull request merged**